scripts/pre-commit.sh

```diff
@@ -45,6 +45,14 @@ function download-dependencies() {
   ./scripts/download-binaries.sh
 }
 
+function run-unit-tests() {
```

Comment:
(No action required) Ugh, I think it's gross that these dependencies are required for unit tests.  I think that requirement should be removed at some point.